server/test/integration/core/docs.spec.ts

```diff
@@ -0,0 +1,94 @@
+import { default as supertest } from 'supertest';
+
+import { App } from '@app';
+import { Config } from '@config/config.types';
+import { default as config } from '@config';
+
+const cfg: Config = config();
+
+describe('[INTEGRATION - CORE] Docs route', () => {
+	describe('Enabled', () => {
+		const api = supertest(new App(false).app);
+
+		it('Should return the swagger documentation', (done) => {
+			api.get('/docs')
+				.redirects(1)
```

Comment:
Waarom gaat die redirecten? 🤔

Reply:
`/docs` redirect in die package naar `/docs/` blijkbaar.